tests/setupTests.js

```diff
@@ -21,6 +21,7 @@ let server;
 const oldWindowLocation = window.location;
 const oldWindowSessionStorage = window.sessionStorage;
 
+jest.retryTimes(3);
```

Comment:
nitpick: a const for the 3?

Reply:
That whole file is basically just a collection of consts 😅 but it's probably as good as opportunity as any to clean that up a bit